doc/api_assets/style.css

```diff
@@ -716,3 +716,19 @@ kbd {
     overflow: hidden;
   }
 }
+@media (prefers-color-scheme: dark) {
+  body {
+    background-color: #090c15; 
```

Comment:
Can you please remove the trailing whitespace?

Reply:
sure thing, 2 minutes